timereport/chalicelib/lib/factory.py

```diff
@@ -11,7 +11,7 @@ def factory(order):
     format_str = "%Y-%m-%d"
     dates, events = [], []
     user_id, user_name = order['user_id'], order['user_name']
-    cmd = order['text'].split("+")
+    cmd = order['text'].split(" ")
```

Comment:
No need to specify empty space since this is default:
```
$ python
Python 3.6.6 (default, Aug 30 2018, 19:46:53)
[GCC 4.2.1 Compatible Apple LLVM 9.1.0 (clang-902.0.39.2)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> "foo bar".split()
['foo', 'bar']
```

Reply:
fixed